src/user/header/table-header.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+
+import { items } from 'user/header/items'
+import { Components } from 'user/header/component'
+
+export const TableHeader = () => <>
```

Comment:
This fragment is redundant. We can remove it.

Reply:
Done

Reply:
I did not mean to remove the component. :smile: 

I meant only the fragment (a.k.a the `<></>` part).

Reply:
Done